### ISSUE TITLE
CI Jenkins jobs always failing

### ISSUE THREAD

**Issue by Neudrino** [magma/magma#11466]
### Describe the Issue

Within the action running on every PR there are (3) Jenkins pipelines. Some of them always fail.
![grafik](https://user-images.githubusercontent.com/13189449/153025789-8cb76b03-98f5-4545-82d6-8fa078239c62.png)

Especially I cannot see any successful run in the following 2 ones.
- https://jenkins.magmacore.org/job/LTE-integration-test/buildTimeTrend
- https://jenkins.magmacore.org/job/magma_integration_tests/buildTimeTrend

The third one mainly fails, however, runs fine occasionally.
- https://jenkins.magmacore.org/job/CWAG-integration-test-libvirt/buildTimeTrend

Checks always failing will make any CI run on respective PRs look always broken. There is not way to judge, if a PR is OK on first sight.
As a developer you would always to have a close look which checks are failing, in order to judge the state of the PR.

### Suggested solution

I would suggest to remove the two always failing Jenkins jobs as checks from Github.

_Optional:_ The third job looks rather flaky and is not required anyway. I suggest to also clean that one up.

Ideally one would not only silence the back channel, but also remove the (webhook) trigger for the respective jobs.

### Additional Context

The suggested actions required admin-permission on Github.

@magma/approvers-ci

**Comment by tmdzk** [comment]
I'll take care of it, thanks @Neudrino for pointing that out

**Comment by just-now** [comment]
>I would suggest to remove the two always failing Jenkins jobs as checks from Github.

It's always easy to switch something off. Still looks like this needs an investigation to understand what amount of coverage magma looses by switching these tests off. If it's 0, probably it's OK except people who are using libvirt-related configuration and tests for development purposes.

**Comment by Neudrino** [comment]
> > I would suggest to remove the two always failing Jenkins jobs as checks from Github.
> 
> It's always easy to switch something off. Still looks like this needs an investigation to understand what amount of coverage magma looses by switching these tests off. If it's 0, probably it's OK except people who are using libvirt-related configuration and tests for development purposes.

The first two jobs I referenced are not providing any coverage, as they are **always** failing. So they provide 0 coverage and it makes no sense to keep them on PRs or linked to Github. In the contrast, they do actually provide a bad developer experience anyone joining the project.
And as I only suggested to disconnect them from Github webhook, if people still need the jobs for very particular test cases (which seemed not to be the case for over 6 month now), they can always trigger those jobs on Jenkins.

Anyway, it is equally easy to turn it on again, if the respective Jenkins jobs got fixed, as this is not a code change, but just a Github admin configuration.
